### ISSUE TITLE
Micro H Brushed Review

### ISSUE THREAD

**Issue by lacymorrow** [lacymorrow/fpv-bible#3]
Spoiler: Shit

**Comment by lazd** [comment]
Yeah almost not worth it. Maybe something more trendy, like we take an review an Eachine inductix clone, note that it's OK but kinda shit, then upgrade it with a Beebrain, retrofit it to use normal Eflite batteries, and build it Shiny Whoop style.

**Comment by lacymorrow** [comment]
Honestly our target demographic is pretty wide, i'm kind of of the opinion that reviewing anything and everything, crap included, is something that will set our content apart.

It's a pretty logical "next step" in FPV to people who don't know better, me and you both made the same mistake, I think we could make it into a pretty interesting conversation about why we thought it was good at the time and why it actually turned out to be shitty.

The more "real" the experience, the better. I imagine our mission statement to be something along the lines of: _"Providing the true, full FPV experience. Vicariously."_

Maybe I'm wrong but I think noobs who want to get in but never will (financial reasons or other) are going to be our primary viewers. The accuracy and personal experience from us will eventually give us a solid rep.

**Comment by lacymorrow** [comment]
Dunno if I'm right, but I think if we can make the format interesting we should do all sorts of topics, not just review the good shit.

**Comment by lacymorrow** [comment]
📋 Reviewed by Paperclip — this repo is legacy/unmaintained. No internal task created.

**Comment by lacymorrow** [comment]
Tracked in Paperclip as [LAC-1192](https://paperclip.ing/LAC/issues/LAC-1192) — stale issue audit for legacy repos.

**Comment by lacymorrow** [comment]
Closing — this repo is no longer actively maintained and this content idea is outdated. Thank you for the contribution.
